torch/testing/_internal/common_nn.py

```diff
@@ -5090,6 +5087,10 @@ def apply_fn(input1, input2, *params):
         # TODO: we don't pass `target` as part of inputs because we don't
         # currently compute the gradient w.r.t. target for loss functions.
         gradcheck(apply_fn, inputs)
+
+        if not self.check_gradgrad:
+            return
+
```

Comment:
nit: it might be better as

```
if self.check_gradgrad:
    gradgradcheck(apply_fn, inputs)
```
that way if someone adds something to the end of this function in the future, they won't have to modify the (if not self.check_gradgrad line) (and they also won't accidentally not read it).

But this is pretty minor and I expect future code changes to get reviewed